Dockerfile

```diff
@@ -37,6 +23,7 @@ ARG GIT_REF
 ARG GIT_BRANCH
 
 COPY package*.json ./
+RUN npm run setup
```

Comment:
Good spot putting this back it.

Looks like it got removed [here](https://github.com/ministryofjustice/hmpps-official-visits-ui/commit/12b8669208f82185d0b16e9a57013f97c09c41d2) by accident.